app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/queue_nav.scala.html

```diff
@@ -16,31 +16,64 @@
-@(queues: Seq[Queue], section: String, caseCountByQueue: Map[String, Int] = Map.empty)(implicit request: AuthenticatedRequest[_], messages: Messages)
+@import uk.gov.hmrc.tariffclassificationfrontend.models.{Permission, Queue, TabIndexes}
+@import uk.gov.hmrc.tariffclassificationfrontend.models.ApplicationType
+@(queues: Seq[Queue], section: String, caseCountByQueue: Map[String, Int] = Map.empty,
+        filteredBy: String = Seq(ApplicationType.BTI,ApplicationType.LIABILITY_ORDER).mkString(","))(implicit request: AuthenticatedRequest[_], messages: Messages)
 
 <div id="queue-navigation" class="side-nav" role="navigation" style="margin-top: 14px">
     <ul>
+
         @if(request.hasPermission(Permission.VIEW_MY_CASES)) {
             <li class="side-nav__list @if(section == "my-cases") { side-nav__list--selected }">
-                <a tabindex="1000" id="nav-menu-my-cases" class="side-nav__link" href="@controllers.routes.MyCasesController.myCases()">@messages("cases.menu.my-cases") (<span id="case-count-my-cases">@caseCountByQueue.getOrElse("my-cases", "0")</span>)</a>
+                <a tabindex="@{TabIndexes.tabIndexForQueue("my-cases")}" id="nav-menu-my-cases" class="side-nav__link" href="@controllers.routes.MyCasesController.myCases()">@messages("cases.menu.my-cases") (<span id="case-count-my-cases">@caseCountByQueue.getOrElse("my-cases", "0")</span>)</a>
             </li>
         }
         @if(request.hasPermission(Permission.VIEW_QUEUE_CASES)) {
-            @for((q,index) <- queues.zipWithIndex) {
-                <li class="side-nav__list @if(section == q.slug) { side-nav__list--selected }">
-                    <a tabindex="@{(index+2) * 1000}" id="nav-menu-queue-@q.slug" class="side-nav__link" href="@controllers.routes.QueuesController.queue(q.slug)">@messages("cases.menu.queue-cases", q.name) (<span id="case-count-@q.slug">@caseCountByQueue.getOrElse(q.id, "0")</span>)</a>
+            @for((q,index) <- queues.zipWithIndex if q.slug == "gateway") {
+                <li class="side-nav__list @if(section == q.slug ) { side-nav__list--selected }">
+                    <a tabindex="@{TabIndexes.tabIndexForQueue(q.slug)}" id="nav-menu-queue-@q.slug" class="side-nav__link" href="@controllers.routes.QueuesController.queue(q.slug)">@messages("cases.menu.gateway-cases") (<span id="case-count-@q.slug">@{caseCountByQueue.getOrElse(s"-${ApplicationType.BTI.toString}", 0)+caseCountByQueue.getOrElse(s"-${ApplicationType.LIABILITY_ORDER.toString}", 0)}</span>)</a>
+                </li>
+            }
+        }
+
+        <li id="nav-menu-header-bti" class="side-nav__list heading">
+            <span>BTI Cases</span>
+        </li>
+        @if(request.hasPermission(Permission.VIEW_QUEUE_CASES)) {
+            @for((q,index) <- queues.zipWithIndex if q.slug != "gateway") {
+                <li class="side-nav__list @if(section == q.slug && filteredBy == "BTI") { side-nav__list--selected }">
```

Comment:
Minor but I'd use the enum rather than String for liability order & bti here

Reply:
Yep, will change that